plugins/services/services_inspector.rb

```diff
@@ -109,11 +109,20 @@ def parse_redhat_chkconfig
       stdout: :capture
     )
 
-    services = output.lines.map do |line|
+    services = output.lines.select { |line|
+        line =~
+        /^\w.*0:(on|off)\s*1:(on|off)\s*2:(on|off)\s*3:(on|off)\s*4:(on|off)\s*5:(on|off)\s*6:(on|off).*$/
+        }.map do |line|
```

Comment:
`end` at 115, 8 is not aligned with `output.lines.select { |line|` at 112, 15 or `services = output.lines.select { |line|` at 112, 4<br>Avoid multi-line chains of blocks.